hail/src/main/scala/is/hail/expr/ir/Interpret.scala

```diff
@@ -886,6 +886,21 @@ object Interpret {
         SafeRow(coerce[PTuple](t), ctx.r, resultOffset).get(0)
       case x: ReadPartition =>
         fatal(s"cannot interpret ${ Pretty(x) }")
+      case TailLoop(name, args, body) =>
```

Comment:
I don't really care if we implement new nodes in Interpret. It's not long for the world.

Reply:
it makes the tests run better for now, so I just threw together a quick thing (we still test against the interpret path for IRSuite by default)